src/main/java/org/jenkinsci/plugins/buildtriggerbadge/LastBuildTriggerColumn.java

```diff
@@ -24,6 +25,25 @@
  * 
  */
 public class LastBuildTriggerColumn extends ListViewColumn {
+	public static final String ICON_ONLY = "iconOnly";
+	public static final String ICON_AND_DESC = "iconAndDesc";
```

Comment:
IMO using an enum here would have been better, but not sure Jenkins's automatic conversion would then work correctly.